core/embed/rust/src/ui/model_tt/component/keyboard/passphrase.rs

```diff
@@ -26,6 +27,14 @@ pub enum PassphraseKeyboardMsg {
     Cancelled,
 }
 
+#[derive(PartialEq, Debug, Copy, Clone)]
+#[cfg_attr(feature = "ui_debug", derive(ufmt::derive::uDebug))]
+enum DisplayStyle {
```

Comment:
That's a good idea, I am just wondering, where to place it.
For the per-model scenario, there is already a common keyboard module, but it will still be repeated multiple times.
For the global (best) one,  I don't see the proper location right away.

Reply:
https://github.com/trezor/trezor-firmware/pull/4385/commits/260468fe83585ce05aa89a2fb4358f0572e5b7bb